internal/n/reliable_notifier.go

```diff
@@ -185,7 +185,6 @@ func NewReliableNotifier(
 		// this logic running (and failing in the background)
 		s.WithRestartTolerance(100, 1*time.Second),
 		s.WithNotifier(notifyRootFailure(settings)),
-		// TODO: Add OneForAll strategy here
```

Comment:
This was a bad idea, the ReliableNotifier doesn't stop receiving events while workers get started, so we lose a lot of events if we restart all the event listeners when one fails